scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/AttestationCLI.scala

```diff
@@ -76,10 +76,16 @@ object AttestationCLI {
 
         val challengeString = params.challenge
         if (challengeString.length() > 0) {
-            val asQuote = as.getQuoteFromServer(challengeString)
-            // System.out.print(asQuote)
+            val asQuote = params.asType match {
+              case ATTESTATION_CONVENTION.MODE_EHSM_KMS =>
+                Base64.getDecoder().decode(as.getQuoteFromServer(challengeString))
+              case ATTESTATION_CONVENTION.MODE_DUMMY =>
```

Comment:
By default, Dummy server return random string for bytes. The format is not correct from SGX's view. I think we can skip for dummy